go-ussd_registration.js

```diff
@@ -1708,14 +1719,16 @@ go.app = function() {
         // PaginatedChoiceState st-05
         self.add('state_last_period_month', function(name) {
             var today = go.utils.get_today(self.im.config);
+            var minimum_weeks = (self.im.config.minimum_weeks || 11);
```

Comment:
This looks in the config for the `minimum_weeks` key, and if it's not there, defaults to 11. I'm not sure if there's a better way of doing a default value in the jsbox apps.